vector-examples.adoc

```diff
@@ -24,7 +24,7 @@ include::example/vvaddint32.s[lines=4..-1]
       add a1, a1, a4              # Bump pointer.
     vmslt.vi v0, v1, 5            # a[i] < 5?
 
-    vsetvli x0, a0, e32,m4.ta,mu  # Vector of 32-bit values.
+    vsetvli x0, x0, e32,m4.ta,mu  # Vector of 32-bit values.
```

Comment:
I do think there is a bug in this line, however:
```suggestion
    vsetvli x0, x0, e32,m4,ta,mu  # Vector of 32-bit values.
```

Reply:
Oops, I tried using this nifty github feature, not realizing it would update the PR rather than creating a new one.  @Sm0ckingBird if it's OK with you, I'm going to close this PR and fix @knightsifive's observation separately.

Reply:
@aswaterman I should have PR-ed this separately, since it was conceptually distinct from this PR. I just got lazy.